### PR TITLE
swi-prolog stable/dev version upgrade

### PR DESCRIPTION
**Pull request by honnix**
stable version upgraded to 7.4.1, dev version upgraded to 7.5.2

- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

**Comment by honnix** [comment]
I think I did something totally wrong for upgrade a stable version. Someone could shed some light?

File: Formula/swi-prolog.rb [hunk 1]
```diff
@@ -1,20 +1,20 @@
...
   revision 2
 
   bottle do
     rebuild 1
-    sha256 "d110d8f643e8ffc1ec03b2803fe35ee4cf0f1fc0b65e1723007496bed928de23" => :sierra
-    sha256 "5d393388c3e50fb3654b6c62245e4803397e0c4a270d6f5efd53b279a779cded" => :el_capitan
-    sha256 "0e8f6ed41a8f7e162752ecf93fbca9ca75faa7787f60736ecceed679699ab07c" => :yosemite
+    sha256 "3e8b6752ba726deef4762205587658a3376ab2ad9fe2d614eebd1b47c1b44062" => :sierra
+    sha256 "7c6562d92d0695286be582c5cd5d39f20fa5f6c3ef66f972fdea628d94fd5fea" => :el_capitan
+    sha256 "0903ea7c9bea4375852424998dea316928ffbc7113b8d27fcc1486ddfdc9ccc6" => :yosemite
```
Comment (fxcoudert) [inline Formula/swi-prolog.rb]: Please do not change the `bottle` block, this is done automatically.

File: Formula/swi-prolog.rb [hunk 2]
```diff
@@ -1,20 +1,20 @@
 class SwiProlog < Formula
   desc "ISO/Edinburgh-style Prolog interpreter"
   homepage "http://www.swi-prolog.org/"
-  url "http://www.swi-prolog.org/download/stable/src/swipl-7.2.3.tar.gz"
-  sha256 "43657d51b7c5887bc2d2bced50a9822b86a08a6841399b8e76ee877f51d646b5"
+  url "http://www.swi-prolog.org/download/stable/src/swipl-7.4.1.tar.gz"
+  sha256 "891e314e8f5d856ef71d8bbce5d255a18b0c8f227628748bb0e1e19473273cc1"
   revision 2
```
Comment (fxcoudert) [inline Formula/swi-prolog.rb]: Please remove `revision 2` line.

**Review by fxcoudert — CHANGES_REQUESTED** [review]

File: Formula/swi-prolog.rb (hunk 2 above)
Comment (honnix) [inline Formula/swi-prolog.rb]: Thanks, found the error in Jenkins.

File: Formula/swi-prolog.rb (hunk 1 above)
Comment (honnix) [inline Formula/swi-prolog.rb]: Sorry about that. I will revert.

**Comment by fxcoudert** [comment]
Thanks @honnix for your contribution!

(Minor comment: next time please use `formula version` as commit message, like `swi-prolog 7.4.1`)

**Comment by honnix** [comment]
Cool. Thanks.

**Pull request closed without merging**